efel/pyfeatures/pyfeatures.py

```diff
@@ -59,6 +61,19 @@ def time():
     return _get_cpp_feature("time")
 
 
+@deprecated("Use spike_count instead.")
+def Spikecount() -> numpy.ndarray:
+    return spike_count()
+
+
+def spike_count() -> numpy.ndarray:
```

Comment:
Is there a reason to have a new `spike_count` feature and deprecating `Spikecount`? Also since it stays as `Spikecount` in the documentation.
Same question for `Spikecount_stimint`

Reply:
In the future I thought keeping Python naming convention would be better. I wanted to deprecate it now so that if we change the naming convention in the future we can say: `this was deprecated 2 major releases ago`.

Reply:
Ideally all features should have a consistent naming. Most of them are underscore separated already e.g.
`adaptation_index` or `ohmic_input_resistance`. `Spikecount` was an exception. What do you think?

Reply:
Alright, sounds good. But then maybe we should update the documentation also.

Reply:
I see `Spikecount` has been used in several places in [BluePyOpt](https://github.com/search?q=repo%3ABlueBrain%2FBluePyOpt%20Spikecount&type=code) and [BluePyEModel](https://github.com/search?q=repo%3ABlueBrain%2FBluePyEModel%20Spikecount&type=code). There can be lot of warnings from old code at multiple places as it is one of the most common features.

Do all the other features follow consistent naming convention? If not, can we keep the name and feature `Spikecount` ?

Reply:
Hi @darshanmandge there is actually a built-in feature in the warnings module to prevent the same warning occurring many times. https://docs.python.org/3/library/warnings.html#the-warnings-filter
When we set its value to `"once"`, the warning filter ensures the repetitive warnings will be shown to the user only once.

> Do all the other features follow consistent naming convention? 

Yes, all of the feature names consisting of two words or more have underscore separation. Only the acronyms like (AP or ADP) have capital letters, rest of the letters are lower and underscore separated. 
E.g. 
AP_peak_upstroke
voltage_deflection_begin
AHP_depth_abs

Reply:
Ok. Thanks!

Reply:
While testing bluepyopt, bluepyefe, bluepyemodel I will make sure this warning do not occur multiple times.

Reply:
Shouldn't you add spike_count  in all_pyfeatures to make it a valid feature? If you want to deprecate Spikecount I mean. Also modify documentation so that feature name is spike_count while keeping a note for Spikecount explaining that it is deprecated.

Reply:
I think "spike_count" is there already.

Reply:
Updating the docs